source/libraries/AbiParser.ts

```diff
@@ -21,7 +21,7 @@ export function parseAbiIntoMethods(ethjsQuery: EthjsQuery, abi: (CompilerOutput
     const result: { [methodName: string]: ContractMethod } = {};
     const items = abi.filter(item => item.type === 'function').forEach(item => {
         result[item.name] = async function(this: TransactionOptions, ...vargs: any[]) {
-            const callConvention = (this.constant || defaultTransaction.constant || item.constant) ? 'call' : 'sendTransaction';
+            const callConvention = (Object.assign({ constant: item.constant }, defaultTransaction, this)).constant ? 'call' : 'sendTransaction';
```

Comment:
This change allows us to do `contract.method.bind({constant: false})(...parameters)` when we want to call a normally constant method on-chain.  Useful for debugging sometimes.